lib/src/modal_page/sliver_wolt_modal_sheet_page.dart

```diff
@@ -0,0 +1,197 @@
+  /// the top bar by focusing specifically on directional navigation within the interface.
+  final double? navBarHeight;
+
+  /// Indicates whether the page height should be at maximum even if the content size is smaller.
+  final bool forceMaxHeight;
+
+  /// A [ScrollController] that controls the scrolling behavior of the page.
+  final ScrollController? scrollController;
+
+  /// A widget representing the action widgets located at the bottom of the page.
+  ///
+  /// The Sticky Action Bar (SAB) guides the user towards the next step. Anchored to the bottom
+  /// of the view, the SAB elevates above the content with a gentle gradient. This position
+  /// guarantees that the action remains visible, subtly hinting to the user that there is more
+  /// content to be explored below the fold.
+  final Widget? stickyActionBar;
+
+  /// Indicates whether a gentle gradient overlay should be rendered above the
+  /// [stickyActionBar]. The purpose of this gradient is to visually suggest
+  /// to the user that additional content might be present below the action bar.
+  ///
+  /// If set to `true`, a gradient from the [sabGradientColor] to transparent is rendered right
+  /// above the [stickyActionBar]. If `false`, no gradient is rendered. By default, it's set to
+  /// `true`.
+  final bool? hasSabGradient;
+
+  /// Controls the draggability of the bottom sheet. This setting overrides the value provided
+  /// via [WoltModalSheet.show] specifically for this page when the modal is displayed as a bottom sheet.
+  final bool? enableDrag;
+
+  /// The color of the gentle gradient overlay that is rendered above the [stickyActionBar]. The
+  /// purpose of this gradient is to visually suggest to the user that additional content might
+  /// be present below the action bar.
+  ///
+  /// If [hasSabGradient] set to `true`, a gradient from this color to transparent is rendered
+  /// right above the [stickyActionBar]. If `false`, no gradient is rendered. By default, it's
+  /// value is to page background color.
+  final Color? sabGradientColor;
+
+  /// A widget representing leading widget in the navigation toolbar. This widget is usually
+  /// a the back button.
+  final Widget? leadingNavBarWidget;
+
+  /// A widget representing trailing widget in the navigation toolbar. This widget is usually
+  /// a the close button.
+  final Widget? trailingNavBarWidget;
+
+  /// Creates a page to be built within [WoltScrollableModalSheet].
+  const SliverWoltModalSheetPage({
+    required this.mainContentSlivers,
+    this.pageTitle,
+    this.navBarHeight,
+    this.topBarTitle,
+    this.heroImage,
+    this.heroImageHeight,
+    this.backgroundColor,
+    this.hasSabGradient,
+    this.enableDrag,
+    this.sabGradientColor,
+    this.forceMaxHeight = false,
+    this.scrollController,
+    this.stickyActionBar,
+    this.leadingNavBarWidget,
+    this.trailingNavBarWidget,
+    this.hasTopBarLayer,
+    this.isTopBarLayerAlwaysVisible,
+    this.topBar,
```

Comment:
`topBar` is aligned below `topBarTitle` but here it is last. 

Align align properties same as constructor for readability.